q-mode.el

```diff
@@ -426,11 +412,10 @@ This marks the PROCESS with a MESSAGE, at a particular time point."
           (fun   (thing-at-point 'sexp))) ; find function body
       (q-send-string (q-strip (concat (buffer-substring start end) fun))))))
 
-(defun q-eval-function-and-go ()
-  "Send the current function to the inferior q[con] process and go."
-  (interactive)
-  (q-eval-function)
-  (q-switch-to-active-buffer))
+(defun and-go (fun) (let ((current-prefix-arg '(16))) (call-interactively fun)))
```

Comment:
``and-go`` leaks into the global namespace.  need to prefix with ``q-`` to become ``q-and-go``.

Reply:
done